ENet-SAD-Simple/datasets/transforms.lua

```diff
@@ -0,0 +1,385 @@
+--
```

Comment:
File is a copy of (or at least very similar to) https://github.com/facebook/fb.resnet.torch/blob/master/datasets/transforms.lua